### DISCUSSION TITLE
How can I use your library to get the original expression (without LaTeX)? And is it possible at the current time?

### DISCUSSION THREAD

**Discussion by dekkab** [ForNeVeR/wpf-math#486] [Q&A] [answered]
```cs
var latex = "\\color[HTML]{82C17A}{\\frac{x^2 + y^2}{z}}";
// ... -> (x^2 + y^2)/z
```

**Comment by ForNeVeR** [comment, marked as answer]
If you are interested in extracting some mathematical value from the markup, then sadly nope, we cannot do this at the moment.

It should be possible to expose the parser internals to convert `\frac{x^2 + y^2}{z}` to something like `frac(sup("x", "2"), sup("y", "2"), "z")`, but so far this is not one of the project's purposes.

I think that the best way to go would be to just fork the parser and introduce the required functionality.

**Reply by dekkab** [reply]
Thank you for your response. As a temporary measure, I will separate my logic into two categories: "raw text" and "TeX text". For now, this approach is easier than developing a full-featured text analysis tool.
